gatsby-browser.js

```diff
@@ -1,10 +1,9 @@
+import './src/styles/fonts.css'
+import './src/styles/global/layout.scss'
 import './src/styles/global/resets.scss'
-
 import './src/styles/global/tokens.scss'
```

Comment:
`tokens.scss` must come before all other CSS files (excepting `fonts.scss`).
Regarding the `global`s, I suggest the following order:
1. `tokens.scss`
2. `resets.scss`
3. `typography.scss`
4. `layout.scss`